packages/platforms/browser/lib/auto-instrumentation/route-change-plugin.ts

```diff
@@ -31,7 +31,9 @@ export class RouteChangePlugin implements Plugin<BrowserConfiguration> {
   configure (configuration: InternalConfiguration<BrowserConfiguration>) {
     if (!configuration.autoInstrumentRouteChanges) return
 
-    let previousRoute = configuration.routingProvider.resolveRoute(new URL(this.location.href))
+    const previousUrl = new URL(this.location.href)
+    let previousRoute = configuration.routingProvider.resolveRoute(previousUrl) || previousUrl.pathname
```

Comment:
Could / should we have a function to do `configuration.routingProvider.resolveRoute(url) || url.pathname` since it's used twice (i.e. also on line 73)?  (An implementation point rather than producty so feel free to ignore!)

Reply:
It should probably just fallback to the `defaultRouteResolver` as otherwise we'd have the same bug if `previousUrl.pathname` was empty